prefect.yaml

```
@@ -0,0 +1,56 @@
+name: prefect-docker-pixi-tiled
+prefect-version: 3.6.0
+
+build:
+  - prefect.deployments.steps.run_shell_script:
+      id: get-commit-hash
+      script: git rev-parse --short HEAD
+      stream_output: false
+  - prefect_docker.deployments.steps.build_docker_image:
+      id: build-image
+      requires: prefect-docker>0.6.0
+      image_name: ghcr.io/nsls2/rsoxs-workflow-prefect3
+      tag: "{{ get-commit-hash.stdout }}"
+      dockerfile: Dockerfile
+      additional_tags:
+        - main
+
+push:
+  - prefect_docker.deployments.steps.push_docker_image:
+      requires: prefect-docker>0.6.0
+      image_name: "{{ build-image.image_name }}"
+      tag: "{{ build-image.tag }}"
+      credentials:
+        "{{
+        prefect.blocks.docker-registry-credentials.ghcr-nsls2-rsoxs-workflow-prefect3
+        }}"
+      additional_tags:
+        - main
+
+pull:
+  - prefect.deployments.steps.set_working_directory:
+      directory: /repo
+  - prefect.deployments.steps.git_clone:
+      repository: https://github.com/JunAishima/workflows-rsoxs.git
+      branch: prefect3-pixi-upgrade
+
+deployments:
+  - name: rsoxs-end-of-run-workflow-docker
+    version: 0.1.0
```

Tags should also be added here pointing at least to the beamline/endstation, but maybe also to the branch, as was done in the HEX updates.
```suggestion
    version: 0.1.0
    tags:
      - rsoxs
      - main
```